src/Program.ad

```diff
@@ -565,6 +565,7 @@ public ParseArgumentsDeclaration(isMainFunction: bool, isMethod: bool) -> string
 	Expect(")");
 	let result: string = arguments.ToString();
 	// TODO implement shortening of String_Builder
+	// Max says: String_Builder has shortening (via `Delete`), but `result` is a string, not a String_Builder...
```

Comment:
`result` is just the `ToString()` of arguments.  The removal of the trailing comma and space should move to the `arguments` `String_Builder`.

Reply:
Done, but see my most recent comment on that code.